frameworks/Go/iris/iris.dockerfile

```diff
@@ -1,14 +1,11 @@
-FROM golang:1.10.1
+FROM golang:latest
```

Comment:
Please keep this at a specific version number

Reply:
Done